### DISCUSSION TITLE
Error running Rector

### DISCUSSION THREAD

**Discussion by gubler** [rectorphp/rector#7662] [Q&A] [answered]
I have rector installed and working on several Symfony projects on my computer. However, when setting up rector for an existing Symfony project today, I get the following error:

> PHP Fatal error:  Uncaught _PHPStan_582a9cb8b\Nette\Schema\ValidationException: Unexpected item 'parameters › deprecationRulesInstalled'. in phar:///Users/dev88/Projects/hive/vendor/rector/rector/vendor/phpstan/phpstan/phpstan.phar/vendor/nette/schema/src/Schema/Processor.php:75

So far I've made sure that I'm using the exact same config as my other applications:

- My computer is running PHP 8.1 (locally, not via Docker), and all applications are using the same installation of PHP
- I'm using Rector v0.15.0 everywhere
- I copied my `rector.php` from my other application
- I have uninstalled and reinstalled rector in my project
- I have run Rector on my other projects to make sure something didn't change on my system
- I added `phpstan/phpstan-deprecation-rules` in my project (none of my other projects include it, but I'm flailing about at this point)

I am lost at this point. I can't see what would be different with this one project where the other 6 projects on my computer are working fine.

Is there any suggestions of where else to look for figuring out what I am doing wrong?

**Comment by TomasVotruba** [comment]
It seems in one of PHPStan configs there is a deprecated opion: "deprecationRulesInstalled"

Remove it or update the packages and you're good to go.

**Reply by gubler** [reply]
I appreciate the quick response :)

Where could this config be?

I have the most up-to-date versions of PHPStan and its plugins I'm using, and I do not have that option in my `phpstan.neon`.

**Comment by gubler** [comment]
OK - this looks like something that is coming from PHPStan 1.9.3 - a new config option that is tripping something up in Rector.

1.9.3 was released this morning and when I rolled back to 1.9.2 and removed `phpstan/phpstan-deprecation-rules` rector started working fine.

**Reply by TomasVotruba** [reply]
This might be related: https://github.com/phpstan/phpstan-deprecation-rules/commit/2c6792eda026d9c474c14aa018aed312686714db

**Comment by samsonasik** [comment, marked as answer]
@gubler you can use latest dev-main which include phpstan 1.9.3
